encoding/m3tsz/encoder.go

```diff
@@ -526,6 +526,11 @@ func (enc *encoder) Stream() xio.SegmentReader {
 	return xio.NewSegmentReader(segment)
 }
 
+func (enc *encoder) StreamLen() int {
+	buff, _ := enc.os.Rawbytes()
```

Comment:
Good call, instead I'll use `OStream Len() int` which already does the `nil` check however.